src/org/usfirst/frc/team1076/robot/subsystems/FrontBackMotors.java

```diff
@@ -27,15 +27,15 @@ public FrontBackMotors(SpeedController frontMotor, SpeedController backMotor) {
 	 * @param speed a number in the range -1 to 1 inclusive.
 	 */
 	public void setFrontSpeed(double speed) {
-		this.frontMotor.set(speed);
+		this.frontMotor.set(speed * 0.5);
 	}
 
 	/**
 	 * Sets the speed of the back motor.
 	 * @param speed a number in the range -1 to 1 inclusive.
 	 */
 	public void setBackSpeed(double speed) {
-		this.backMotor.set(speed);
+		this.backMotor.set(speed * -0.5);
```

Comment:
Yeah, that's a good idea. Do you have any idea why `motor.reverseOutput(true)` wouldn't work on the `CANTalon`? It didn't seem to fix the problem, which is why we have to rely on this.